### PR TITLE
Fix DarkReader doesn't handle images correctly

### PR DESCRIPTION
**Pull request by quanglam2807**
Resolve https://github.com/atomery/webcatalog/issues/993

### CODE REVIEW

File: public/preload/view.js [hunk 1]
```diff
@@ -3,43 +3,59 @@
...
+  const darkReader = ipcRenderer.sendSync('get-preference', 'darkReader');
+  let loadScript = '';
+  if (shouldUseDarkColor && darkReader) {
+    const {
+      darkReaderBrightness,
+      darkReaderContrast,
+      darkReaderGrayscale,
+      darkReaderSepia,
+    } = ipcRenderer.sendSync('get-preferences');
+    console.log(`DarkReader.setFetchMethod(window.fetch);
```
Comment (swrobel) [inline public/preload/view.js:24]: Seems like you could just do this after the following and `console.log(loadScript)`

Comment (quanglam2807) [reply public/preload/view.js:24]: Thank you, Stefan! I actually rewrote it here: https://github.com/atomery/juli/pull/92/files

Comment (swrobel) [reply public/preload/view.js:24]: OK, so how do I get it to update the Juli version to 9.6.0? 
Screen Shot 2020-08-30 at 2 05 45 PM

Comment (quanglam2807) [reply public/preload/view.js:24]: It’s up now :)

**Pull request merged**